src/welldev/io/model/ProductList.java

```diff
@@ -0,0 +1,37 @@
+package welldev.io.model;
+
+public class ProductList {
```

Comment:
Class name Product list is not appropriate as model class. You could use only product and collection of products can be product list.